xbmc/video/dialogs/GUIDialogAudioSubtitleSettings.cpp

```diff
@@ -204,7 +204,7 @@ void CGUIDialogAudioSubtitleSettings::OnSettingAction(const CSetting *setting)
       strPath = g_application.CurrentFileItem().GetPath();
 
     std::string strMask = ".utf|.utf8|.utf-8|.sub|.srt|.smi|.rt|.txt|.ssa|.aqt|.jss|.ass|.idx|.rar|.zip";
-    if (g_application.GetCurrentPlayer() == "videoplayer")
+    if (g_application.GetCurrentPlayer() == "VideoPlayer")
```

Comment:
when I worked on this https://github.com/xbmc/xbmc/pull/8764 I noticed some string comparisons with "videoplayer" and I have promised myself to check when I had a time if there was some typo, so I found these two cases

but sincerely atm I do not know much about this one specific subject